deploy/osd-managed-resources/get-managed-resources.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+if [ -f  /tmp/output.log ]; then
+    rm -f /tmp/output.log
+fi
```

Comment:
I get that `/tmp` is meant to be temporary, but this is alarming to see. This script doesn't know the history of this file

I'd recommend the use of `mktemp(1)`, or including the functionality of this script into the Python script so there's no need for a temporary file.

Reply:
Thanks for the review. I've updated the script to be entirely in python and removed any dependency on a temp file.